src/components/level-indicator/level-indicator.js

```diff
@@ -34,14 +34,14 @@ class LevelIndicator extends React.PureComponent {
             height: '8px',
             borderRadius: '1px'
           }}
-          className={`inline-block w6 bg-${squareColor} align-middle mr3`}
+          className={`w6 bg-${squareColor} mr3`}
```

Comment:
I noticed the `inline-block` class on this line and 42 were unneeded. Also, `align-middle`.